networking/src/test/java/google/registry/networking/handler/SslInitializerTestUtils.java

```diff
@@ -67,21 +66,26 @@ public static KeyPair getKeyPair() throws Exception {
   }
 
   /**
-   * Signs the given key pair with the given self signed certificate to generate a certificate with
+   * Signs the given key pair with the given self-signed certificate to generate a certificate with
    * the given validity range.
    *
    * @return signed public key (of the key pair) certificate
    */
   public static X509Certificate signKeyPair(
-      SelfSignedCaCertificate ssc, KeyPair keyPair, String hostname, Date from, Date to)
+      SelfSignedCaCertificate ssc, KeyPair keyPair, String hostname, DateTime from, DateTime to)
       throws Exception {
     X500Name subjectDnName = new X500Name("CN=" + hostname);
     BigInteger serialNumber = BigInteger.valueOf(System.currentTimeMillis());
     X500Name issuerDnName = new X500Name(ssc.cert().getIssuerX500Principal().getName());
     ContentSigner sigGen = new JcaContentSignerBuilder("SHA256WithRSAEncryption").build(ssc.key());
     X509v3CertificateBuilder v3CertGen =
         new JcaX509v3CertificateBuilder(
-            issuerDnName, serialNumber, from, to, subjectDnName, keyPair.getPublic());
+            issuerDnName,
+            serialNumber,
+            from.toDate(),
```

Comment:
This is literally still using a `java.util.Date` then, only it's purposely being packaged into a DateTime, and then stripped out of it, for no reason!